packages/dev/core/src/Materials/GreasedLine/greasedLinePluginMaterial.ts

```diff
@@ -260,8 +293,8 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase implements IGr
             const activeCamera = this._scene.activeCamera;
 
             if (activeCamera) {
-                const projection = activeCamera.getProjectionMatrix();
-                uniformBuffer.updateMatrix("grl_projection", projection);
+                uniformBuffer.updateMatrix("grl_projection", activeCamera.getProjectionMatrix());

```

Comment:
```suggestion
                uniformBuffer.updateMatrix("grl_projection", this._scene.getProjectionMatrix());
```
To be consistent with the next line, and because you can now get rid of the check on `activeCamera`.

Reply:
Absolutely makes sense. `activeCamera` check removed and the `projectionMatrix` is set from the `scene` object.